src/util/records.py

```diff
@@ -106,6 +106,36 @@ def remove_view(cr, xml_id=None, view_id=None, silent=False, key=None):
         for [v_id] in cr.fetchall():
             remove_view(cr, view_id=v_id, silent=silent, key=xml_id)
 
+    # Occurrences of xml_id in the t-call of views are to be found and removed.
+    if xml_id != "?":
+        arch_col = (
+            get_value_or_en_translation(cr, "ir_ui_view", "arch_db")
```

Comment:
Sadly, this will only return `en_US`, so all other translations will be still broken. I've seen elsewhere that we don't care much about languages. @aj-fuentes do you think it would make sense to modify the usage of this method to return all available languages? Or create a new one that would do that?

Reply:
`edit_view` will use en_US anyway, so it makes sense for the query to select the candidates in the base arch, when one language doesn't really match the base arch (en_US) then it's too specific for us to care about